### PR TITLE
Feature/gate of knowledge control creatures

### PR DESCRIPTION
**Pull request by Phobossus**

### CODE REVIEW

File: RP-Core/src/main/java/abilitiesV2/skills/abitilies/knowledge/gateofknowledge/wishes/ControlCreaturesWish.java [hunk 1]
```diff
@@ -0,0 +1,291 @@
...
+    }
+
+    private Effect getControlCreaturesEffect(LivingCharacter caster) {
+        SpecialCommandEffect takeControlEffect = new SpecialCommandEffect.Builder(SpecialCommandEffect.Type.CAN_B_COMMAND)
+                .applyActions(() -> {
+                    EffectMessage.send(caster,
+                            "You have received the power to control other creatures.",
+                            EffectMessageAudience.HOLDER);
+                })
+                .usageMessage(String.format("/b control - to take control of creatures in your %s radius.", TAKE_CONTROL_RADIUS))
```
Comment (Pedrokase) [inline RP-Core/src/main/java/abilitiesV2/skills/abitilies/knowledge/gateofknowledge/wishes/ControlCreaturesWish.java:87]:
```suggestion
                .usageMessage(String.format("/b control - to take control of creatures in a %s block radius.", TAKE_CONTROL_RADIUS))
```

File: RP-Core/src/main/java/abilitiesV2/skills/abitilies/knowledge/gateofknowledge/wishes/ControlCreaturesWish.java [hunk 2]
```diff
@@ -0,0 +1,291 @@
...
+                                return selected;
+                            })
+                            .limit(MAX_CONTROLLED_PER_TAKE_CONTROL_ORDER)
+                            .collect(Collectors.toList());
+                    if (!selectedCreatures.isEmpty()) {
+                        EffectMessage.send(caster,
+                                "You attempt to take control of the nearby creatures.",
+                                EffectMessageAudience.HOLDER);
+                        EffectMessage.send(caster,
+                                String.format("%s gains control over the nearby creatures.",
```
Comment (Pedrokase) [inline RP-Core/src/main/java/abilitiesV2/skills/abitilies/knowledge/gateofknowledge/wishes/ControlCreaturesWish.java:112]:
```suggestion
                                String.format("%s gains control over nearby creatures.",
```

File: RP-Core/src/main/java/abilitiesV2/skills/abitilies/knowledge/gateofknowledge/wishes/ControlCreaturesWish.java [hunk 3]
```diff
@@ -0,0 +1,291 @@
...
+                            .filter(selected -> TargetingUtils.isInClearPath(selected, caster, TAKE_CONTROL_RADIUS))
+                            .map(selected -> {
+                                shootControlMissiles(caster, selected);
+                                return selected;
+                            })
+                            .limit(MAX_CONTROLLED_PER_TAKE_CONTROL_ORDER)
+                            .collect(Collectors.toList());
+                    if (!selectedCreatures.isEmpty()) {
+                        EffectMessage.send(caster,
+                                "You attempt to take control of the nearby creatures.",
```
Comment (Pedrokase) [inline RP-Core/src/main/java/abilitiesV2/skills/abitilies/knowledge/gateofknowledge/wishes/ControlCreaturesWish.java:109]: If the character succeeds, why output "attempted"? 
Also, remove "the"
```suggestion
                                "You take control of nearby creatures.",
```

Comment (Phobossus) [reply RP-Core/src/main/java/abilitiesV2/skills/abitilies/knowledge/gateofknowledge/wishes/ControlCreaturesWish.java:109]: Ye old message prior to refactor that has a separate msg for failure

**Review by Pedrokase — APPROVED** [review]

**Pull request merged**